src/index.js

```diff
@@ -0,0 +1,167 @@
+// 'use strict';
+import 'regenerator-runtime/runtime';
+import axios from 'axios';
+
+const state = {
```

Comment:
Very excellent idea to create a global state JSON variable. 👍🏻 As we've seen in react, this effectiveness translates to state variables in components!